apps/extension/public/manifest.json

```diff
@@ -11,9 +11,16 @@
   "content_scripts": [
     {
       "matches": ["<all_urls>"],
-      "js": ["js/content-scripts.js"],
+      "js": ["js/injected-connection-manager.js"],
       "run_at": "document_start",
       "all_frames": true
+    },
+    {
+      "matches": ["<all_urls>"],
+      "js": ["js/expose-penumbra-global.js"],
+      "run_at": "document_start",
+      "all_frames": true,
+      "world": "MAIN"
```

Comment:
I think we should either have the `install-detector.txt` method _or_ the injected script method. Is there a good reason why we'd prefer injecting the script over fetching the txt file?